object/oidc_discovery.go

```diff
@@ -160,3 +172,39 @@ func GetJsonWebKeySet() (jose.JSONWebKeySet, error) {
 
 	return jwks, nil
 }
+
+func GetWebFinger(resource string, rels []string, host string) (WebFinger, error) {
+	wf := WebFinger{}
+
+	resourceSplit := strings.Split(resource, ":")
+
+	if len(resourceSplit) != 2 {
+		return wf, fmt.Errorf("invalid resource")
+	}
+
+	resourceType := resourceSplit[0]
+	resourceValue := resourceSplit[1]
+
+	oidcDiscovery := GetOidcDiscovery(host)
+
+	switch resourceType {
+	case "acct":
+		user, err := GetUserByEmailOnly(resourceValue)
+		if err != nil {
+			return wf, err
+		}
+
+		if user == nil {
+			return wf, fmt.Errorf("user not found")
+		}
+
+		wf.Subject = resource
+
+		wf.Links = append(wf.Links, WebFingerLink{
+			Rel:  "http://openid.net/specs/connect/1.0/issuer",
```

Comment:
according to [standard document of webfinger](https://www.rfc-editor.org/rfc/pdfrfc/rfc7033.txt.pdf), rel should not be `"http://openid.net/specs/connect/1.0/issuer"`

![image](https://github.com/user-attachments/assets/40b03eba-115a-46b9-9e00-fb249de1dbde)

Reply:
you should test this function first